test/e2e/traceflow_test.go

```diff
@@ -53,9 +56,14 @@ func skipIfTraceflowDisabled(t *testing.T, data *TestData) {
 	}
 }
 
+var (
+	protocolTCP    = int32(6)
```

Comment:
Since these protocol variables are provided, could you please also replace protocol number in IPv4 cases? Like L115.

Reply:
Moved all protocol to these variables. Thanks.